service/build.gradle

```diff
@@ -221,14 +234,16 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'send-letter-client', version: '2.5.0'
   compile group: 'uk.gov.service.notify', name: 'notifications-java-client', version: '3.15.1-RELEASE'
 
-  compile group: 'com.google.guava', name: 'guava', version: '29.0-jre'
-  compile group: 'commons-io', name: 'commons-io', version: '2.7'
+  implementation group: 'com.google.guava', name: 'guava', version: '29.0-jre'
+  implementation group: 'commons-io', name: 'commons-io', version: '2.7'
 
-  compile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
+  implementation group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
 
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-mail'
 
-  testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: versions.springBoot) {
+  testImplementation group: 'org.apache.pdfbox', name: 'pdfbox', version: '2.0.19'
+
+  testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test') {
```

Comment:
Also is there a reason for the removal of `version: versions.springBoot`

Reply:
removed this to take advantage of the spring dependency management